src/CommunityToolkit.Maui.CameraView/CommunityToolkit.Maui.CameraView.csproj

```diff
@@ -0,0 +1,77 @@
+    <RepositoryUrl>https://github.com/CommunityToolkit/Maui</RepositoryUrl>
+    <PackageReleaseNotes>See https://github.com/CommunityToolkit/Maui/releases</PackageReleaseNotes>
+    <DefineConstants>$(DefineConstants);</DefineConstants>
+    <UseFullSemVerForNuGet>false</UseFullSemVerForNuGet>
+    <Title>.NET MAUI CameraView</Title>
+    <Description>CameraView is a view for displaying a preview of the camera output and other camera related functionalities in your .NET MAUI app.</Description>
+    <PackageIcon>icon.png</PackageIcon>
+    <Product>$(AssemblyName) ($(TargetFramework))</Product>
+    <AssemblyVersion>1.0.0.0</AssemblyVersion>
+    <AssemblyFileVersion>1.0.0.0</AssemblyFileVersion>
+    <Version>1.0.0-pre1</Version>
+    <PackageVersion>$(Version)$(VersionSuffix)</PackageVersion>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <PackageTags>dotnet,maui,toolkit,kit,communitytoolkit,dotnetcommunitytoolkit,cameraview,photo</PackageTags>
+    <Configurations>Debug;Release</Configurations>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="..\..\build\nuget.png" PackagePath="icon.png" Pack="true"/>
+    <None Include="ReadMe.txt" pack="true" PackagePath="."/>
+  </ItemGroup>
+
+  <ItemGroup Condition="$(TargetFramework.Contains('-android'))">
+    <PackageReference Include="Xamarin.AndroidX.Camera.View" Version="1.3.1.1" />
+    <PackageReference Include="Xamarin.AndroidX.Camera.Camera2" Version="1.3.1.1" />
+
+    <!-- Xamarin.AndroidX.Camera 1.3.1.1 depends on Xamarin.AndroidX.Lifecycle.LiveData 2.7.0.1:
+    https://www.nuget.org/packages/Xamarin.AndroidX.Camera.Core/#dependencies-body-tab 
+    However, Dotnet MAUI still implicitly references Xamarin.AndroidX.Lifecycle.LiveData 2.6.1.3:
+    https://github.com/dotnet/maui/blob/main/eng/AndroidX.targets
+    This causes build error "Detected package downgrade":
+    https://learn.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1605
+    Workaround: add a package reference to the higher version of the package. Possibly can be removed when MAUI bumps the implicit reference version.
+    -->
+    <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.7.0.1" />
+
+    <!--Adding the above package reference requires Workaround for https://github.com/xamarin/AndroidX/issues/800#issuecomment-1774639627 -->
+    <!-- Tracking issue https://github.com/xamarin/AndroidX/issues/764 -->
+    <PackageReference Include="Xamarin.AndroidX.Collection" Version="1.4.0.1" />
+    <PackageReference Include="Xamarin.AndroidX.Collection.Ktx" Version="1.4.0.1" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Maui.Controls" Version="$(MauiPackageVersion)"/>
+    <PackageReference Include="Microsoft.Maui.Controls.Compatibility" Version="$(MauiPackageVersion)"/>
+    <PackageReference Include="CommunityToolkit.Mvvm" Version="8.2.2" />
```

Comment:
I don't think we should be bringing in a dependency on the MVVM toolkit here.

Reply:
Agreed, if we actually need the property values to change lets do it manually, lets not take an extra dependency here